docker/pcap-miner/Dockerfile

```diff
@@ -0,0 +1,9 @@
+
+FROM demisto/python3:3.8.3.8715
+
+COPY requirements.txt .
+
+RUN apk --update add --no-cache --virtual .build-dependencies python-dev build-base wget git \
+  && pip install --no-cache-dir -r requirements.txt \
+  && apk del .build-dependencies
+  && apk install tshark
```

Comment:
@yaakovi added this line (opened a demisto/python3:3.8.3.8715 docker and saw that it installs tshark 3.0.2). Is this the correct place to add it?